examples/with-jest/package.json

```diff
@@ -15,9 +15,8 @@
     "babel-jest": "^18.0.0",
     "enzyme": "^2.5.1",
     "jest-cli": "^18.0.0",
-    "react": "^15.3.2",
```

Comment:
I think react and react-dom were added as peerDep in recent commit. You should not remove this.